general-configuration-guide/use-tinymce-classic.md

```diff
@@ -8,13 +8,13 @@ keywords: theme classic
 
 {{site.productname}} has three main integration modes:
 
-* "classic" form-based
-* inline editing
-* distraction-free
+* "classic" form-based mode
```

Comment:
"form-based"? that doesn't mean anything to me tbh

Reply:
I think it's meant to mean for use in HTML forms, but I agree it's confusing and we should remove it or replace it with something clearer.

Reply:
oh good. I didn't like it either. Changing to `"classic" full editor mode.`